huxley/core/middlewares.py

```diff
@@ -12,3 +12,11 @@ class ServerLoggingMiddleware(object):
     def process_exception(self, request, exception):
         logger = logging.getLogger('huxley.server')
         logger.exception(exception)
+
+class LoggingMiddleware(object):
+    def process_response(self, request, response):
+        if request.path.find('api') >= 0:
+            logger = logging.getLogger('huxley.db')
```

Comment:
This suggests it's a database logger, which doesn't make sense. Why not make it `huxley.api`?